src/App.svelte

```diff
@@ -117,6 +127,10 @@
     await sendCommand('SetStudioModeEnabled', { studioModeEnabled: !isStudioMode })
   }
 
+  async function toggleReplay () {
+    await sendCommand('ToggleReplayBuffer', { outputActive: !isReplaying })
```

Comment:
When users don't have replay buffer enabled, this returns an error: `Replay buffer is not available.`. Should be somehow reflect that in the UI?